app/views/export/generic.html.erb

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+    .divider:nth-child(odd) {
+      border-top: thick double;
+      border-bottom: thick double;
+      background-color: #e3e3e3;
+    }
+  </style>
+  <title><%= "test" %></title>
+</head>
+<body>
+<% id_count = nil %>
+<% @data.each do |data| %>
+  <% if id_count == nil %>
+    <% id_count = data[0].to_i.truncate %>
```

Comment:
Some of the ids are like '5.2' as well as 5, so we have to truncate it.

As a side note: thinking it through, I probably should of called id_count into something like 'previous_major_id' or something like that to make it clearer.